src/openms/source/ANALYSIS/QUANTITATION/IsobaricIsotopeCorrector.cpp

```diff
@@ -40,16 +39,21 @@ namespace OpenMS
 
     Matrix<double> correction_matrix = quant_method->getIsotopeCorrectionMatrix();
 
-    if (matrixIsIdentityMatrix(correction_matrix))
+    if (correction_matrix.getEigenMatrix().isIdentity(0.0))
     {
-      throw Exception::InvalidParameter(__FILE__, __LINE__, OPENMS_PRETTY_FUNCTION,
-                                        "IsobaricIsotopeCorrector: The given isotope correction matrix is an identity matrix leading to no correction. "
-                                        "Please provide a valid isotope_correction matrix as it was provided with the sample kit!");
+      OPENMS_LOG_DEBUG << "Correction matrix is the identity matrix." << std::endl;
+      OPENMS_LOG_DEBUG << correction_matrix << std::endl;
+
+      // workaround: TMT11plex has a special case where the correction matrix is the identity matrix
+      if (quant_method->getMethodName() != "tmt11plex")
```

Comment:
I think we should at least warn. I think this was intended to force users to put their matrix in there.

Reply:
do you have any idea why this did not fail before? I don't see it...